spec/value_objects/challenged_decision_status_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+RSpec.describe ChallengedDecisionStatus do
```

Comment:
It is just so the coverage doesn't complain. We are also doing it for intents and dispute_types.
The other option is to add a :nocov: block.